### ISSUE TITLE
Misspelled words are not highlighted

### ISSUE THREAD

**Issue by auzias** [atom/spell-check#81]
Hi, first of all, thanks for your nice work!

When I enable [language-latex](https://atom.io/packages/language-latex) for my LaTeX projects, misspelled words are not highlighted any more.

Is there a way around this? Did someone experienced it?

A [cross-bug report has been made](https://github.com/area/language-latex/issues/55).

**Comment by izuzak** [comment]
Check out the readme: https://github.com/atom/spell-check#spell-check-package-

> By default spell check is enabled for the following files: Plain Text, GitHub Markdown, Git Commit Message. You can override this from the Spell Check settings in the Settings view (`cmd-,`).

Also see https://github.com/atom/spell-check/issues/79#issuecomment-135899610 for help on finding the name of the scope.

**Comment by auzias** [comment]
Thanks a lot!

I totally agree with [#79 (comment)](https://github.com/atom/spell-check/issues/79#issuecomment-135899610), you made a very well work, but there is a wee lack of guidance on this :wink:

Anyway, thank for your fast and rightful answer!
Have a nice day

**Comment by izuzak** [comment]
Agreed -- I'll see if I can get those instructions merged today. Sorry for the confusion!

**Comment by lalomartins** [comment]
Note, I just found the hard way — if a theme or user style sets a background colour for cursor-line, which [piatto-light](https://atom.io/packages/piatto-light) does, highlight will disappear for the current line, because `.highlight .region` has `z-index: -1;`. I have no idea where this definition comes from, grep couldn't find it :grinning: but it's there on a fresh profile. I worked around it in my user stylesheet by overriding it specifically for spell-check:

```css
// Spell-check highlight has z-index -1 which makes it disappear under cursor-line
atom-text-editor::shadow .spell-check-misspelling .region {
  z-index: 0;
}
```

**Comment by andrewfowlie** [comment]
tldr - the scope for latex is called `text.tex.latex`
